lua/lspconfig/pses.lua

```diff
@@ -0,0 +1,45 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "pses"
+local temp_path = vim.fn.stdpath('cache')
+
+local function make_cmd(bundle_path)
+  if bundle_path ~= nil then
+    local command = bundle_path .. [[/PowerShellEditorServices/Start-EditorServices.ps1 -BundledModulesPath ]] .. bundle_path .. [[ -LogPath ]] .. temp_path .. [[/pses.log -SessionDetailsPath ]] .. temp_path .. [[/pses.session.json -FeatureFlags @() -AdditionalModules @() -HostName nvim -HostProfileId 0 -HostVersion 1.0.0 -Stdio -LogLevel Normal]]
```

Comment:
This is a little messy, string.format might be a better choice with this many substitutions.

Reply:
I can fix this